shoonyatech/frontend.social
# [WIP][DO NOT MERGE] feat/zoom initial setup for zoom

STEPS TO TEST:
1. Go to https://zoom.us/ & create a meeting
2. Generate API keys from zoom (https://marketplace.zoom.us/) 
    - Click on Develop dropdown 
    - Then click on Build App option
    - Generate JWT token
3. Check out the code and replace the API key and API secret key in **`Zoom.html`**
4. Navigate to the event detail page
5. There you will notice 3 input boxes
6. Enter your alias, meeting id(from the meeting you have created) and the meeting password
